documentation/source/models.md

```diff
@@ -0,0 +1,105 @@
+## Model Zoo
+
+SuperGradients provides an extensive collection of state-of-the-art (SOTA) models in its [model zoo](http://bit.ly/3EGfKD4).
+These models are implemented as `torch.nn.Module` and can be used, customized, and trained like any other torch module.
+
+The 3 main use cases of the Model Zoo are to
+- Train a model from scratch
+- Fine-tune a pre-trained model
+- Use a model (pre-trained or not) as the backbone of a larger architecture.
+
+### I. Instantiating a model
+
+To instantiate a model, specify the model name and the number of classes desired.
+```python
+from super_gradients.training import models
+
+# Instantiate resnet18 with head supporting 100 classes
+default_resnet18 = models.get(model_name="resnet18", num_classes=100)
+```
+
+Every model name is accessible in to [model zoo](http://bit.ly/3EGfKD4),but can also be dynamically accessed through `super_gradients.common.object_names` for autocompletion
```

Comment:
please rephrase: 
All model names are available in the [model zoo]...

Reply:
Changed in 0774f85754e17e396c6c377aeca58cdf8b888dd7